src/components/Campaign.tsx

```diff
@@ -43,6 +52,12 @@ export const Campaign = () => {
     campaignContentMutation.mutate({ idCampaign, content });
   };
 
+  const nextUrl =
+    searchParams.get("redirectedFromSummary")?.toUpperCase() === "TRUE"
+      ? `${dopplerLegacyBaseUrl}/Campaigns/Summary/Index?IdCampaign=${idCampaign}`
+      : `${dopplerLegacyBaseUrl}/Campaigns/Recipients/Index?IdCampaign=${idCampaign}
+      &RedirectedFromSummary=False&RedirectedFromTemplateList=False`;
+
```

Comment:
❤️

Reply:
Cómo se renderiza eso? no se introduce un salto de línea?

Si es así, tal vez convenga hacer algo parecido a esto:

```suggestion
  const nextUrl =
    searchParams.get("redirectedFromSummary")?.toUpperCase() === "TRUE"
      ? `${dopplerLegacyBaseUrl}/Campaigns/Summary/Index?IdCampaign=${idCampaign}`
      : `${dopplerLegacyBaseUrl}/Campaigns/Recipients/Index?IdCampaign=${idCampaign}`
        + `&RedirectedFromSummary=False&RedirectedFromTemplateList=False`;
```

o

```suggestion
  const nextUrl =
    searchParams.get("redirectedFromSummary")?.toUpperCase() === "TRUE"
      ? `${dopplerLegacyBaseUrl}/Campaigns/Summary/Index?IdCampaign=${idCampaign}`
      : `${dopplerLegacyBaseUrl}/Campaigns/Recipients/Index?IdCampaign=${idCampaign}${
        '' }&RedirectedFromSummary=False&RedirectedFromTemplateList=False`;
```

Reply:
Confirmado:

![image](https://user-images.githubusercontent.com/1157864/164503204-4c743198-1a33-4232-aecf-d79254bcc921.png)

No parece una buena idea poner saltos de línea y espacios dentro del literal.

Reply:
![image](https://user-images.githubusercontent.com/45947909/164506885-3aba8b23-9599-4778-9f99-45a658891685.png)
No veo que renderice salto de línea

Reply:
Será porque al modificar el DOM el browser lo arregla porque es robusto.

Robusto significa ser liberal con las entradas y estricto con las salidas.

Seamos robustos nosotros también y no descansemos esa responsabilidad en el browser.

Reply:
Me gusta la primera opción que propone @andresmoschini

Reply:
Fixed